src/tests/EdgeID/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod edge_tests;
+pub mod transition_id_tests;
```

Comment:
add eol to this file, some lister is complaining

Reply:
Maybe "cargo fmt"

Reply:
fixed